spells.yml

```diff
@@ -10305,6 +10305,23 @@ witch_bolt:
     - sorcerer
     - warlock
     - wizard
+wizards_strike:
+  casting_time: 1 standard action
+  components: S, M
+  concentration: false
+  description: You swipe the spell's material component in the direction of a foe to bash it from a distance. Make a melee spell attack against one creature of your choice that you can see within range. On a hit, the target takes 2d10 bludgeoning, piercing, or slashing damage (your choice).
```

Comment:
Upon watching the video I myself linked to, this would have to push the target (maybe Medium or smaller) 5 ft (more with higher level?) to stay true to the video. That might be too strong, though

Reply:
That also kind of just looks like _thunderwave_.

Reply:
I don't think that's a bad thing. Aside from the damage type being different, one would be targeted, from a distance, and overall more consistent (attack roll > saving throw), while the other is close-range AOE (and loud).

Giving it a control effect makes sense for a wizard too, but it should probably be 2d8 damage in that case, or maybe even 1d10